vision:
  ballhsv:
    h0: 0
    h1: 180
    s0: 0
    s1: 115
    v0: 110
    v1: 255
  camera:
    devNumber: 0
    flipHor: 0
    flipVer: 0
    heightUnDistortion: 1177
    useBagfile: 1
    widthUnDistortion: 1620
  dataAssociation:
    MaxAngDiff: 45
    MaxErrorForInlier: 30
    _MaxAngleChange: 10
    _MinCompLengthForModelError: 30
  debug:
    publishTime: 0
    saveProcessingTime: 0
    saveTrajectory: 1
    showAssociateLines: 0
    showBoundingRects: 1
    showCamPose: 0
    showCorrespondence: 1
    showFieldHull: 1
    showFieldModel: 0
    showHoughLines: 0
    showLineInfo: 0
    showMarkers: 1
    showMergedLines: 0
    showModelLines: 0
    showNodeGraph: 1
    showRobPose: 1
    showSkeletonPixels: 1
    showTangentLine: 0
    showTrajectory: 1
    useKFCovType0: 1
    useKalmanFilter: 1
    useMotionOdom: 1
    useParticleFilter: 0
    useRansac: 0
    useSolvePnP: 0
    useYawSpeed: 1
    visTrajectoryData: 0
  field:
    BonnField: 1
    FieldBorder: 0
    MaxDistanceFromBottomOfImage: 200
    maxContourCount: 3
    minArea: 600
  fieldhsv:
    h0: 37
    h1: 77
    s0: 107
    s1: 219
    v0: 36
    v1: 255
  goal:
    DistanceToMerge: 20
    MaxOutField: 40
    MinLineLength: 30
    VoteDouble: 18
    cannyThreadshold: 21
    dilate: 0
    erode: 0
    jumpMax: 25
    minContinuesColor: 10
    minDoubleLength: 3
    minEdgeValue: 100
  goalhsv:
    h0: 24
    h1: 126
    s0: 0
    s1: 255
    v0: 120
    v1: 255
  graphNode:
    AngleToMerge: 5
    DistanceCompToMerge: 0
    LineWidth: 15
    MaxDist: 0
    MaxDistanceForMergeNode1: 50
    MaxDistanceForMergeNode2: 50
    MaxLineSegDistance: 35
    MaxLineSegDistanceForJoint: 400
    MaxProjectedDistance: 5
    MergeJointDist: 10
    MinCmpLineLength: 30
    MinDist: 1000
    MinDistForMerge: 10
    Min_adj: 5000
    SamplePointDist: 3
    SmallAngle: 63
    _AvgAngleDiff: 15
    _MaxAngleDiff: 15
    _MaxAvgProjectedDistance: 10
    _MaxCloseNum: 15
    _MaxDistanceForConnect: 40
    _MaxDistanceForMerge: 80
    _MaxLineSegDistance: 20
    _MaxProjectedDistance: 15
    _MinAngle: 45
    _MinAngleForSameComp: 159
    _MinCmpLineLength: 30
    _Min_adj: 1
    _NeighborRadius: 15
  hillclimbing:
    ErrorThreshold1: 5
    ErrorThreshold2: 50
    HeadingOffset: 0
    MaxHypothesisNum: 60
    MaxIterationNum: 5
    MovingStepCoefficient: 0.02
  icp:
    AvgDistThresholdForConf: 3
    HeadingOffset: 3.1416
    IcpInlierDistThreshold: 4912
    InlierNumThresholdForConf: 30
    KFOrientationCov: 10000
    KFPositionCov: 10000
    SamplePointDist_WorldCord: 0.2
  kalmanFilter:
    ErrorThreshod: 5
    KFOrientationCov: 1
    KFPositionCov: 1
    KFSpeedX: 0
    KFSpeedY: 0
    KFSpeedYaw: 0
  line:
    AngleToMerge: 25
    AngleToMergeImg: 3
    Div1: 0.1
    Div2: 0.3
    HoughLineMinLineLength: 28
    HoughLineRho: 7
    HoughLineTheta: 6
    HoughLineVoteThreshold: 19
    HoughLinemaxLineGap: 70
    LocalOptimalRange: 0
    MaxNeighborDist: 5
    MaxPointLineDistance: 100
    MeasAreaThreshod: 60000
    MeasLineLengthThreshod: 200
    MeasLineMaxAngDifThreshod: 60
    MeasLineNumThreshod: 4
    MinBrightnessValue: 28
    MinLineLength: 80
    MinLineSegDistance: 0.8
    MinLineSegDistanceImg: 53
    MinProjectedDistance: 0.35
    MinProjectedDistanceImg: 23
    MinSkeletonValue: 75
    ObstacleLineDistance: -20
  linehsv:
    h0: 0
    h1: 180
    s0: 11
    s1: 255
    v0: 0
    v1: 254
  location:
    x: -1.425
    y: 0
    z: 0.578
  obstacle:
    minArea: 30
  obstaclehsv:
    h0: 0
    h1: 4
    s0: 0
    s1: 4
    v0: 0
    v1: 11
  orientation:
    x: 0
    y: 0
    z: 0
  particleFilter:
    HeadingOffset: 0
    MinSumOfLineLengthOnImage: 100
    ParticleNum: 200
    RandomSamplePct: 0.2
  projection:
    timeToShift: 0.150
  square:
    MaxNeighborDist: 6
    Max_K: 12
    Max_K_border: 4
    MinPointNum: 1
